todo/forms.py

```diff
@@ -0,0 +1,22 @@
+from django import forms
+from django.core.exceptions import ValidationError
+
+from todo.models import Tag, Task
+
+
+class TagForm(forms.ModelForm):
+    class Meta:
+        model = Tag
+        fields = "__all__"
+
+
+class TaskForm(forms.ModelForm):
+    tags = forms.ModelMultipleChoiceField(
+        queryset=Tag.objects.all(),
+        widget=forms.CheckboxSelectMultiple,
+        required=False
+    )
+
+    class Meta:
+        model = Task
+        fields = "__all__"
```

Comment:
We should add validation if passed deadline is in the future. To add validation to the form we can add clean method, which should be named in convention `clean_fieldname`. To check for future date we can use timezone from `django.utils`

```suggestion
        fields = "__all__"
        
        def clean_deadline(self):
            deadline = self.cleaned_data.get('deadline')
            if deadline and deadline <= timezone.now():
                raise ValidationError('The deadline must be in the future.')
            return deadline
```

ref. [custom form validation](https://stackoverflow.com/questions/7948750/custom-form-validation)